tests/network/test_reaction_network.py

```diff
@@ -55,17 +55,11 @@ def test_characterize_path(self):
 
         # run calc
         path_instance = ReactionPath.characterize_path(
-            path,
-            "softplus",
-            RN.min_cost,
-            RN.graph,
-            solved_PRs,
-            RN.PR_byproducts,
-            PR_paths,
+            path, "softplus", RN.graph, solved_PRs
         )
 
         # assert
-        self.assertEqual(path_instance.byproducts, [356, 182, 548])
+        self.assertEqual(path_instance.byproducts, [356, 548])
```

Comment:
I'm assuming that we've lost a byproduct here because we stopped checking PR path byproducts. IMO let's change it back, and in that case I'm assuming that 182 will return as a byproduct for this test